doc/source/gallery/geopandas_rasterio_sample.ipynb

```diff
@@ -0,0 +1,303 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "# Using GeoPandas with Rasterio to sample point data\n",
+    "\n",
+    "This example shows how to use GeoPandas with Rasterio.  [Rasterio](https://rasterio.readthedocs.io/en/latest/index.html) is a package for reading and writing raster data.\n",
+    "\n",
+    "In this example a set of vector points is used to sample raster data at those points.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import geopandas\n",
+    "import rasterio\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Load the vector data\n",
+    "=============================\n",
+    "\n",
+    "Let's open a geopackage of the points we want to use to sample the raster data\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gdf = geopandas.read_file('data\\\\sample_points.gpkg')"
```

Comment:
This file does not exist. The example needs to be fully reproducible. I'd suggest creating points manually.

Reply:
Sure, no problem.  Points now created manually.